MAUI/ProgressBar/custom-content.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: Custom Content in .NET MAUI ProgressBar control | Syncfusion
+description: Learn here all about customcontent support in Syncfusion .NET MAUI ProgressBar control, its elements and more.
+platform: MAUI
+control: ProgressBar
+documentation: ug
+---
+
+# Custom Content in .NET MAUI ProgressBar (Progress Bar)
+
+In the circular progress bar, you can add any view to the center using the `Content` property. 
```

Comment:
In the circular progress bar, add any view to the center using the `Content` property.

Reply:
done